main/options.c

```diff
@@ -2215,9 +2218,9 @@ static void processEchoOption (const char *const option, const char *const param
 static void processForceQuitOption (const char *const option __unused__,
 				    const char *const parameter)
 {
-	long s = 0;
-	if (parameter != NULL && parameter[0] != '\0')
-		s = strtol (parameter, NULL, 0);
+	long s;
+	if (parameter == NULL || parameter[0] == '\0' || !strToLong(parameter, 0, &s))
+		s = 0;
 	exit (s);
```

Comment:
Now there is `strToInt()` it would probably better to use an int argument to exit() as it's what it expects.  Not that it changes much, but it'd be cleaner.